src/code/actions.js

```diff
@@ -275,9 +363,33 @@ export function initiateDelayedFertilization(delay, gamete1, gamete2) {
   };
 }
 
-export function keepOffspring() {
+export function _keepOffspring(index, success) {
```

Comment:
FWIW: When I see a function name starting with an underscore `_`, I assume it isn't an externally available function, but `export` proves me wrong here.